components/brave_rewards/browser/database/database_unblinded_token.cc

```diff
@@ -16,6 +16,13 @@
 
 namespace brave_rewards {
 
+namespace  {
+
+const char* table_name_ = "unblinded_tokens";
```

Comment:
why are these two lines needed as we have it already defined in h file

Reply:
They were defined as class instance members, so they got allocated for every 'DatabaseUnblindedToken' instance. 
As they should be 'static' - Chromium standards say to put them in the .cc 
Also, I needed to reference 'table_name_' in my new static class method.

Reply:
based on the fact that we do this in all other DB files, can you revert here and change them all in follow up?

Reply:
Sure, I don't mind - though the change was instigated by needing `table_name_` in a static method, so that would force other changes (none of them monumental). 

I could revert this and duplicate the `table_name_` to get us by - but this is taking us down the wrong route IMHO and I'd rather move the right way. But, your call, I'm happy either way.

Reply:
sounds good, let's keep it like this for now. Please log an issue to migrate all other db files to this new approach